<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>657</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1051</width>
    <height>657</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>657</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NotePad++</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>484</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>484</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Editor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
      <widget class="QWidget" name="widget">
       <attribute name="title">
        <string>Calendar</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCalendarWidget" name="calendarWidget">
          <property name="minimumSize">
           <size>
            <width>976</width>
            <height>437</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>976</width>
            <height>437</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>72</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>72</height>
       </size>
      </property>
      <property name="title">
       <string>CONTROLS</string>
      </property>
      <widget class="QDial" name="dial">
       <property name="geometry">
        <rect>
         <x>920</x>
         <y>20</y>
         <width>50</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>8</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>30</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>40</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Word Count</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>10</y>
         <width>20</width>
         <height>61</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>10</y>
         <width>21</width>
         <height>61</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>40</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>40</y>
         <width>32</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>LineNumbering</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionRedo"/>
    <addaction name="actionUndo"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionNotePad_Pro"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menuNotePad_Pro">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionFont_Color"/>
    <addaction name="actionFont_Background"/>
    <addaction name="separator"/>
    <addaction name="actionBOLD"/>
    <addaction name="actionItalics"/>
    <addaction name="actionUnderline"/>
    <addaction name="separator"/>
    <addaction name="actionSubscript"/>
    <addaction name="actionSuperscript"/>
    <addaction name="separator"/>
    <addaction name="actionAlign_Text_Left"/>
    <addaction name="actionAlign_Text_Right"/>
    <addaction name="actionCenter"/>
    <addaction name="actionJustify"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionClear_All_Formating"/>
    <addaction name="actionClear_All_Text"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionImage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuNotePad_Pro"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menuFormat"/>
   <addaction name="menuAdd"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionFont_Color"/>
   <addaction name="actionFont_Background"/>
   <addaction name="separator"/>
   <addaction name="actionBOLD"/>
   <addaction name="actionItalics"/>
   <addaction name="actionUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionSubscript"/>
   <addaction name="actionSuperscript"/>
   <addaction name="separator"/>
   <addaction name="actionAlign_Text_Left"/>
   <addaction name="actionAlign_Text_Right"/>
   <addaction name="actionCenter"/>
   <addaction name="actionJustify"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_All"/>
   <addaction name="actionClear_All_Formating"/>
   <addaction name="actionClear_All_Text"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>34</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>34</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
   <addaction name="actionUndo"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="minimumSize">
    <size>
     <width>715</width>
     <height>34</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>715</width>
     <height>34</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImage"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/new.png</normaloff>:/rec/img/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/open.png</normaloff>:/rec/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/save_file_disk_open_searsh_loading_clipboard_1513 (1).png</normaloff>:/rec/img/save_file_disk_open_searsh_loading_clipboard_1513 (1).png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/save_as.jpeg</normaloff>:/rec/img/save_as.jpeg</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/cut.png</normaloff>:/rec/img/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/copy.jpg</normaloff>:/rec/img/copy.jpg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/paste.png</normaloff>:/rec/img/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/redo (2).png</normaloff>:/rec/img/redo (2).png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/undo.png</normaloff>:/rec/img/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionNotePad_Pro">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/info.png</normaloff>:/rec/img/info.png</iconset>
   </property>
   <property name="text">
    <string>NotePad++</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/print.png</normaloff>:/rec/img/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/log-in.png</normaloff>:/rec/img/log-in.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/font.png</normaloff>:/rec/img/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionFont_Color">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/text.png</normaloff>:/rec/img/text.png</iconset>
   </property>
   <property name="text">
    <string>Font Color</string>
   </property>
  </action>
  <action name="actionFont_Background">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/highlight.png</normaloff>:/rec/img/highlight.png</iconset>
   </property>
   <property name="text">
    <string>Font Background</string>
   </property>
  </action>
  <action name="actionSubscript">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/subscript.png</normaloff>:/rec/img/subscript.png</iconset>
   </property>
   <property name="text">
    <string>Subscript</string>
   </property>
  </action>
  <action name="actionSuperscript">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/pngegg.png</normaloff>:/rec/img/pngegg.png</iconset>
   </property>
   <property name="text">
    <string>Superscript</string>
   </property>
  </action>
  <action name="actionAlign_Text_Left">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/align-left.png</normaloff>:/rec/img/align-left.png</iconset>
   </property>
   <property name="text">
    <string>Align Text Left</string>
   </property>
  </action>
  <action name="actionAlign_Text_Right">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/right-align.png</normaloff>:/rec/img/right-align.png</iconset>
   </property>
   <property name="text">
    <string>Align Text Right</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/center-align.png</normaloff>:/rec/img/center-align.png</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/justification.png</normaloff>:/rec/img/justification.png</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
  </action>
  <action name="actionClear_All_Formating">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/eraser.png</normaloff>:/rec/img/eraser.png</iconset>
   </property>
   <property name="text">
    <string>Clear All Formating</string>
   </property>
  </action>
  <action name="actionClear_All_Text">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/remove.png</normaloff>:/rec/img/remove.png</iconset>
   </property>
   <property name="text">
    <string>Clear All Text</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/add-image.png</normaloff>:/rec/img/add-image.png</iconset>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/selection.png</normaloff>:/rec/img/selection.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionBOLD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/bold.png</normaloff>:/rec/img/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionItalics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/italics.png</normaloff>:/rec/img/italics.png</iconset>
   </property>
   <property name="text">
    <string>Italics</string>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/underline.png</normaloff>:/rec/img/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
